testhelpers/testhelpers.go

```diff
@@ -184,12 +184,26 @@ func PushImage(dockerCli dockercli.CommonAPIClient, ref string) error {
 	return rc.Close()
 }
 
-func HTTPGetE(url string) (string, error) {
-	resp, err := http.DefaultClient.Get(url)
+func HTTPGetE(url string, headers ...map[string]string) (string, error) {
+	client := http.DefaultClient
+
+	request, err := http.NewRequest("GET", url, nil)
 	if err != nil {
-		return "", err
+		return "", errors.Wrap(err, "making new request")
+	}
+
+	if len(headers) > 0 {
+		for key, val := range headers[0] {
+			request.Header.Set(key, val)
+		}
+	}
+
+	resp, err := client.Do(request)
```

Comment:
```suggestion
	resp, err := http.DefaultClient.Do(request)
```
If this is the only place we use the client why not inline like this and remove the `client` var?